package/libyui.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Nov 29 14:38:54 UTC 2019 - Rodion Iafarov <riafarov@suse.com>
+
+- Add support to operate on many widgets with rest-api
+- 3.9.0
```

Comment:
Please also mention the so_version 11 bump.

Reply:
Done. Thanks!